### ISSUE TITLE
Goto buttons not setting the active class

### ISSUE THREAD

**Issue by roscopeeco** [kswedberg/jquery-carousel-lite#29]
Hi

First of all got to say GREAT PLUGIN! The original was good but you've added in some great functionality.

I've come across an issue with the active class not being set when using the goto buttons.

I was supplying an array of goto selectors.

After a lot of head scratching and moving between different jquery versions with no luck i managed to resolve the issue by modifying line 78 in the source from

```
    $btnsGo = $(o.btnGo);
```

to:
        $btnsGo = $(o.btnGo.join());

So basically it's just supplying a string instead of an array.

Thanks

Ross
